index.ts

```diff
@@ -248,14 +258,10 @@ async function injectContentScriptInSpecificTarget(
 			files: script.js ?? [],
 			matchAboutBlank: script.matchAboutBlank ?? script.match_about_blank,
 			runAt: script.runAt ?? script.run_at,
-		}),
+		}, options),
 	]);
 
-	if (ignoreTargetErrors) {
-		await catchTargetInjectionErrors(Promise.all(injections));
-	} else {
-		await Promise.all(injections);
-	}
```

Comment:
This was unnecessary because this logic already exists in the specific injectors